cors.go

```diff
@@ -251,7 +255,21 @@ func AllowedOriginValidator(fn OriginValidator) CORSOption {
 	}
 }
 
-// ExposeHeaders can be used to specify headers that are available
+// OptionStatusCode sets a custom status code on the OPTIONS requests.
+// Default behaviour sets it to 200 to reflect best practices. This is option is not mandatory
+// and can be used if you need a custom status code (i.e 204) or have a monitoring
```

Comment:
Remove the piece about monitoring middleware - as per my other comment, there are more robust ways to get the real status code in middleware when logging.

Reply:
Totally agree we should keep this generic. I'm removing it.